lib/ansible/template/__init__.py

```diff
@@ -610,56 +621,9 @@ def __init__(self, loader, shared_loader_obj=None, variables=None):
         self.cur_context = None
 
         self.SINGLE_VAR = re.compile(r"^%s\s*(\w*)\s*%s$" % (self.environment.variable_start_string, self.environment.variable_end_string))
-
-        self._clean_regex = re.compile(r'(?:%s|%s|%s|%s)' % (
-            self.environment.variable_start_string,
-            self.environment.block_start_string,
-            self.environment.block_end_string,
-            self.environment.variable_end_string
-        ))
```

Comment:
Note that this is also being removed in https://github.com/ansible/ansible/pull/68560

Reply:
The above PR has been merged and I rebased this PR so this changed is no longer here.